docs/docs/topics/auth.mdx

```diff
@@ -29,14 +21,62 @@ Infrahub supports two authentication methods
 | infrahubctl        | Soon | Yes   |
 | GraphQL Playground | No   | Yes   |
 
-More information on managing API token can be found in the [managing API tokens guide](/guides/managing-api-tokens).
+More information on managing API tokens can be found in the [Managing API Tokens Guide](/guides/managing-api-tokens).
 
 :::info
 
 While using the API, the authentication token must be provided in the header:
 
 ```yaml
-X-INFRAHUB-KEY: 06438eb2-8019-4776-878c-0941b1f1d1ec
-```
+X-INFRAHUB-KEY: 06438eb2-8019-4776-878c-0941b1f1d1ec```
 
 :::
+
+## Assigning permissions
```

Comment:
Should we keep the permission part in "authentication" topic? Shouldn't we create a new "authorization" or similar topic? or move it to permission/roles topic

Reply:
I hesitated for this one, I think it was important to bring the gap between auth and permissions here, specially with the `default setup`, even if the permissions explanation are in the other file.

Could have a link from permission to here as well 🤔